### DISCUSSION TITLE
How to convert `Value` to unquoted string

### DISCUSSION THREAD

**Discussion by ndarilek** [async-graphql/async-graphql#1127] [Q&A]
Is there an easier way to convert `async_graphql` instances to strings that isn't:

```
            if let GraphQLValue::String(username) = &**username {
```

Not that that's particularly difficult, but it does do quite a bit of nesting if I have several value types that I'm trying to convert to strings.

For reference, I'm working with arbitrary untyped payloads of GraphQL values and a non-public API. If one of the string conversions fails, panicking is fine.

`to_string` appears to quote the value if it's a string, which I don't want. There appears to be a `From` impl for `ConstValue`, but I'm not clear on how to convert to this nor does it seem to be available. I could get away with using ConstValue in this case since I don't need variables here.

Thanks.

**Comment by sunli829** [comment]
No, but you can implement it yourself. 🙂
